### PR TITLE
[TECH] Modal window

### PR DESCRIPTION
**Pull request by kseniyaLine**
modal-window with material-ui

### CODE REVIEW

File: src/components/modal/Modal.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+  return {
+    top: `${top}%`,
+    left: `${left}%`,
+    transform: `translate(-${top}%, -${left}%)`,
+  };
+}
+
+// eslint-disable-next-line no-undef
+const useStyles = makeStyles((theme) => ({
```
Comment (fshovchko) [inline src/components/modal/Modal.js]: Use Modal.scss for styling

**Review by tensenrey — COMMENTED** [review]
please change name of pull request on [TECH] modal window

File: package.json [hunk 2]
```diff
@@ -64,6 +64,7 @@
     ]
   },
   "devDependencies": {
+    "@lingui/core": "^3.10.4",
```
Comment (mtalstykh-exadel) [inline package.json]: it isn't part of your ticket. Remove the line

File: src/App.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import React from 'react';
-import Login from './pages/login/login';
-import { Switch, Route } from 'react-router-dom';
+import SimpleCounter from './store/store';
```
Comment (mtalstykh-exadel) [inline src/App.js:2]: why do you import it here?

File: package.json [hunk 4]
```diff
@@ -3,25 +3,25 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@lingui/react": "^3.10.2",
     "@material-ui/core": "^4.12.0",
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
+
```
Comment (mtalstykh-exadel) [inline package.json:12]: empty strings...

**Pull request closed without merging**